### PR TITLE
JAMES-3396 Prevent RRT loops upon Alias, Forward, Address Mapping and Group creation

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Review by LanKhuat — APPROVED** [review]

File: server/protocols/webadmin/webadmin-data/src/test/java/org/apache/james/webadmin/routes/ForwardRoutesTest.java [hunk 1]
```diff
@@ -238,6 +240,27 @@ void putUserInForwardShouldReturnNoContent() {
...
         }
 
+        @Test
+        void putShouldDetectLoops() {
+            with()
+                .put(ALICE + SEPARATOR + "targets" + SEPARATOR + BOB);
+
+            Map<String, Object> errors = when()
+                .put(BOB + SEPARATOR + "targets" + SEPARATOR + ALICE)
+                .then()
```
Comment (Arsnael) [inline server/protocols/webadmin/webadmin-data/src/test/java/org/apache/james/webadmin/routes/ForwardRoutesTest.java:250]: indent?

**Review by Arsnael — APPROVED** [review]

**Comment by chibenwa** [comment]
`[ERROR] /james-project/server/container/util/src/main/java/org/apache/james/util/DataChunker.java:[27,30] package com.google.common.base does not exist`

**Review by Arsnael — APPROVED** [review]

**Comment by chibenwa** [comment]
test this please

**Review by nvduc91 — APPROVED** [review]

**Comment by Arsnael** [comment]
Merged

**Pull request closed without merging**